src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/inspectors/PlayerPerformanceInspector.java

```diff
@@ -27,34 +28,63 @@ public class PlayerPerformanceInspector implements PerformanceInspector {
      */
     private final UUID uuid;
 
+    /**
+     * The order type of the timings.
+     */
+    private final SummaryOrderType orderType;
+
     /**
      * This creates a new {@link PlayerPerformanceInspector} for the given {@link Player}.
      * 
      * @param player
      *            The {@link Player}
+     * @param orderType
+     *            The order type of the timings
      */
-    public PlayerPerformanceInspector(@Nonnull Player player) {
+    public PlayerPerformanceInspector(@Nonnull Player player, @Nonnull SummaryOrderType orderType) {
```

Comment:
This is also a breaking change if we can add a constructor that has a default orderType, can just be the normal when not actually adding any flag

You can also use the `@ParametersAreNonnullByDefault` here